src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/amp.test.js.snap

```diff
@@ -231,17 +231,6 @@ Object {
 }
 `;
 
```

Comment:
Oh dear, don't think this was supposed to be removed...?

Reply:
There are a number of snapshots which have changed, and some have even been removed - would it be worth checking why (I don't think they should have changed as part of this piece of work).

Reply:
This is still in progress :)

Reply:
This is quite strange - I've updated the snapshots again after finishing with my changes I even got more of them removed from other page types.

Reply:
😢 Not sure - is this branch up-to-date with latest?

Reply:
It is - I'm gonna revert them and see if CI passes and ignore them in local.

Reply:
@itimirichard is also getting updates on the `onDemandTVPage` snapshots. I wonder if they should have been updated in this PR https://github.com/bbc/simorgh/pull/6282.

Reply:
Think it should have been https://github.com/bbc/simorgh/pull/6733

Reply:
Yes, if the `onDemandTVPage` snapshots updates should have been done in this PR #6282 but was skipped, then we can leave them in the IDX integretion PR #6802

Reply:
I believe this has now been fixed - updating to latest will resolve.

Reply:
I keep getting snapshots updates for the `mediaAssetPage` and `storyPage`, which I didn't commit and I don't understand why. I wonder if they should had been updated in previous PRs too.

<img width="559" alt="Screenshot 2020-06-16 at 09 29 35" src="https://user-images.githubusercontent.com/46446236/84750967-e9847400-afb3-11ea-8802-215087d90090.png">